list.js

```diff
@@ -23,14 +23,25 @@ class List {
     let newDiv = document.createElement('div')
     newDiv.setAttribute("id", `list_${list.id}`)
     newDiv.setAttribute('class', 'list')
+    newDiv.setAttribute('data-id', list.id)
+
+    let button = document.createElement('button')
+    button.innerText = "X"
+
     let title = document.createElement('h2')
-    title.innerText = list.title
+    title.append(button)
```

Comment:
`.append` is a way to add content to the existing content inside the tag. Thats what we were looking for earlier.